### PR TITLE
Pablo/new analytics events

### PR DESCRIPTION
**Pull request by azagaya**

### CODE REVIEW

File: src/html_shell/godot.html [hunk 1]
```diff
@@ -145,6 +145,21 @@
 			return(os);
 		}
 
+		function getBrowser() {
+			let ua = navigator.userAgent
+			let browser = "Unknown"
+
+			if (ua.indexOf("Seamonkey") >= 0) browser = "Seamonkey";
+			else if (ua.indexOf("Firefox") >= 0) browser = "Firefox";
+			else if (ua.indexOf("Chromium") >= 0) browser = "Crhomium";
```
Comment (RPicster) [inline src/html_shell/godot.html:154]: I think there is a typo at "Crhomium"

File: src/html_shell/godot.html [hunk 2]
```diff
@@ -145,6 +145,21 @@
...
+			else if (ua.indexOf("Chrome") >= 0) browser = "Crhome";
```
Comment (RPicster) [inline src/html_shell/godot.html:155]: Also maybe a typo? "Crhome"

File: src/scripts/ui_main.gd [hunk 3]
```diff
@@ -7,6 +7,34 @@ func _ready() -> void:
 	_g.connect("ui_scale_decrease", self, "ui_scale_down")
 	SaveLoadSettings.connect("settings_loaded", self, "_on_settings_loaded", [], CONNECT_ONESHOT)
 	SaveLoadSettings.load_settings()
+	
+	# Check errors of previous session
```
Comment (RPicster) [inline src/scripts/ui_main.gd:11]: Maybe I missed it - but shouldn't the local storage variable "error" be cleared after sending this Error event?

**Review by RPicster — CHANGES_REQUESTED** [review]

File: src/scripts/ui_main.gd (hunk 3 above)
Comment (azagaya) [reply src/scripts/ui_main.gd:11]: I moved it below making the post

**Review by RPicster — APPROVED** [review]

**Pull request merged**